docs/develop/terrad/commands.mdx

```diff
@@ -1,6 +1,6 @@
 # Commands
 
-This section describes the commands available from Terrad, the command line interface that connects a running Terrad process.
+This section describes the commands available on Terrad, the command line interface that connects to a running Terrad process.
```

Comment:
This should Probably be changed so we don't define Terrad using Terrad. 

"This section describes the commands available on Terrad, the Terra blockchain's command line interface and node daemon."